### ISSUE TITLE
BEActivateBuffer

### ISSUE THREAD

**Issue by ivanrvpereira** [spf13/spf13-vim#194]
I have this error when i switch of windows 

```
Error detected while processing function <SNR>49_BEActivateBuffer:
line   18:
E684: list index out of range: 2    
```

Already tried to make a clean install.

**Comment by jamestomasino** [comment]
I get this error a lot when opening multiple tabs in MacVim. I haven't been able to figure out what triggers it, or how to get around it.

**Comment by jairzh** [comment]
yes, i get the some error

**Comment by jimmydburr** [comment]
ditto. bump.

**Comment by goswarm** [comment]
I'm guessing one of the plugins is borking.  Try eliminating each by commenting out and restarting MacVim... laboriously testing.  Please post your results here...

**Comment by ivanrvpereira** [comment]
I already tested it, and is the bufexplorer plugin that gives that errors.
I'm using without bufexplorer and until now, no more errors.

**Comment by jamestomasino** [comment]
I've added a [wiki page](https://github.com/spf13/spf13-vim/wiki/Bufexplorer) with a method for disabling the plugin in your .vimrc.local.

**Comment by spf13** [comment]
I think I'm just going to remove it entirely. It's not so much that it's
problematic itself, as much as it conflicts with so many other plugins
which utilize buffers.

## 

Steve Francia
http://stevefrancia.com
http://spf13.com
http://twitter.com/spf13

**Comment by jimmydburr** [comment]
Steve, yes, removal gets my vote.

**Comment by spf13** [comment]
I've removed it https://github.com/spf13/spf13-vim/commit/bc0affbf1c5ad0ce32d4deed7ab044ee63de37c3

**Comment by michaellenahan** [comment]
Hi there, I installed spf13-vim today. The installation experience was great and I'm looking forward to using it. I've been using vim for about a year or so.

One thing I noticed was that the BufExplorer mapping is still in the .vimrc, even though BufExplorer is not available.

line 387
nmap <leader>b :BufExplorer<CR>

... so <leader>b gives us this error:
E492: Not an editor command: BufExplorer

I recommend either removing the mapping or re-purposing it.

In my case, I'm using this mapping in my .vimrc.local to allow quick buffer switching.
nnoremap <leader>b :buffers<CR>:buffer<Space>

... this keeps the <leader>b mapping and allows us to type a number to go to the buffer we want.

The original tip came from here:
http://vim.wikia.com/wiki/Easier_buffer_switching
